modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb

```diff
@@ -21,7 +21,9 @@ def perform(power_of_attorney_id, form_number, rep_id)
         rep = ::Veteran::Service::Representative.where(representative_id: rep_id).order(created_at: :desc).first
 
         output_path = pdf_constructor(form_number).construct(data(power_of_attorney, form_number, rep),
-                                                             id: power_of_attorney.id)
+                                                             id: power_of_attorney.id,
+                                                             created_at: power_of_attorney.created_at)
+
```

Comment:
Could we instead have the `data` method merge an attribute like `dateSigned` or `appointmentDate => power_of_attorney.created_at`? This way, we do not have to thread the `created_at` param through the constructors and can avoid signatures like `def page1_options(data, _created_at = nil)`.

Reply:
@tycol7 Ya great idea, that worked really well, [77c144a](https://github.com/department-of-veterans-affairs/vets-api/pull/18803/commits/77c144a579a2c6192b1cef089edd99dd8cc8578a) (Ignore the gmeil file change reverted it in follow up)